userspace/libsinsp/sinsp.cpp

```diff
@@ -684,6 +689,11 @@ void sinsp::open_modern_bpf(unsigned long driver_buffer_bytes_dim, uint16_t cpus
 	params.buffer_bytes_dim = driver_buffer_bytes_dim;
 	params.cpus_for_each_buffer = cpus_for_each_buffer;
 	params.allocate_online_only = online_only;
+    params.kernel_config_path = NULL;
+    if (!kernel_config_path.empty())
+    {
+        params.kernel_config_path = kernel_config_path.c_str();
+    }
```

Comment:
[nit] Indentation got a little out of hand here
```suggestion
	params.kernel_config_path = NULL;
	if (!kernel_config_path.empty())
	{
		params.kernel_config_path = kernel_config_path.c_str();
	}
```